### ISSUE TITLE
A screencast

### ISSUE THREAD

**Issue by mrezauli** [gregsullivan/_tw#8]

**Comment by gregsullivan** [comment]
Am I right to understand this as a documentation request, in the form of screencasts?

More video content is certainly on my roadmap, but there are a few other features I'd like to add to _tw before further expanding the documentation (especially with screencasts, where updating them to incorporate changes and new features would be non-trivial).

**Comment by mrezauli** [comment]
Yep! That's great. I think screencast is better way to describe something.

**Comment by gregsullivan** [comment]
You're not the first to tell me as much! I'm hopeful this is something I can do at some point during the spring, but it very much depends on the schedules for my client work.

I'll leave this issue open, and I'll post here as I have progress to report.

**Comment by mrezauli** [comment]
Thanks

**Comment by gregsullivan** [comment]
I'm hoping to close this issue in the first half of 2026! I have two projects completing soon that will work well as screencasts, and I'm planning to record the process of building them out.
